src/invoker_cache.cpp

```diff
@@ -44,13 +45,27 @@ boost::optional<const Invoker&> InvokerCache::GetFound1_0(const std::string& net
                                                           const std::string& algorithm) const
 {
     const auto item = invokers.find(network_config);
-    if(item == invokers.end() || item->second.found_1_0.empty())
+    if(item == invokers.end())
+    {
+        MIOPEN_LOG_I2("No invokers found for " << network_config);
+        return boost::none;
+    }
+    if(item->second.found_1_0.empty())
+    {
+        MIOPEN_LOG_I2("Invokers found for " << network_config
+                                            << " but there is no find 1.0 result.");
```

Comment:
do we want to hard-code this find 1.0 here?

Reply:
Yes. This will go away when we switch to Find 2.0 API. @DrizztDoUrden can explain the details.

Reply:
Are you speaking about the info messages? I haven't changed anything around except for them.